CompetitionTemplate/src/competition/subsystems/arm/arm_commands/ArmManualControlCommand.java

```diff
@@ -26,7 +26,24 @@ public void initialize() {
 
     @Override
     public void execute() {
-        armSubsystem.setArmMotorPower(-oi.operatorJoystick.getVector().y/2);
+        double desire = -oi.operatorJoystick.getVector().y;
```

Comment:
"desire" doesn't help a reader understand what the variable does. This should be changed to something more specific.